python/GafferSceneUI/LightEditor.py

```diff
@@ -440,6 +440,37 @@ def __toggleBoolean( self, inspections ) :
 
 		return True
 
+	def __activeInspectionTweaks( self, pathListing ) :
+
+		selection = pathListing.getSelection()
+
+		columns = pathListing.getColumns()
+
+		tweaks = []
+
+		with Gaffer.Context( self.getContext() ) as context :
+			for i in range( 0, len( columns ) ) :
```

Comment:
We could ditch all the `[i]` indexing if we did this :

```
for selection, column in zip( pathListing.getSelection(), pathListing.getColumns() ) :
```

Might make the inner loop a tiny bit more readable?

Reply:
`zip` is new to me, I like it! Fixed in 3b1e785f90f9c76297491c2fa26e5e17f799f77e along with a few other spots.